conkit/command_line/conkit_plot.py

```diff
@@ -331,6 +350,24 @@ def altloc_remove(cmap):
         figure = conkit.plot.ContactMapChordFigure(con_sliced, use_conf=args.confidence, legend=True)
         figure_aspect_ratio = 1.0
 
+    elif args.which == 'contact_map_matrix':
+
+        seq = conkit.io.read(args.seqfile, args.seqformat)[0]
+        con = conkit.io.read(args.confile, args.conformat)[0]
+
+        con.sequence = seq
+        con.assign_sequence_register()
+
+        if args.otherfile:
+            other = conkit.io.read(args.otherfile, args.otherformat)[0]
+            other.sequence = seq
+            other.assign_sequence_register()
+        else:
+            other = con
+
```

Comment:
I think you could recode this to:

`if args.otherfile:
    other = conkit.io.read(args.otherfile, args.otherformat)[0]
else:
    other = con

other.sequence = seq
other.assign_sequence_register()`

and remove lines 358 and 359.

Reply:
Actually scratch this, you're plotting them both next to each other so I can see why your format con first.